test/transcoding/SPV_INTEL_joint_matrix/joint_matrix_bfloat16.ll

```diff
@@ -0,0 +1,217 @@
+; RUN: llvm-as < %s -o %t.bc
+
+; RUN: llvm-spirv -s %t.bc -o %t.regularized.bc
+; RUN: llvm-dis %t.regularized.bc -o %t.regularized.ll
+; RUN: FileCheck < %t.regularized.ll %s --check-prefix=CHECK-REGULARIZED
+
+; RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_bfloat16_conversion,+SPV_INTEL_joint_matrix -o %t.spv
+; RUN: llvm-spirv %t.spv -to-text -o %t.spt
+; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
+
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+; RUN: llvm-dis < %t.rev.bc | FileCheck %s --check-prefix=CHECK-LLVM
+
+; CHECK-REGULARIZED: %spirv.JointMatrixINTEL._bfloat16_8_16_0_3
+; CHECK-REGULARIZED: %ref.tmp.i = alloca %"class.cl::sycl::ext::intel::experimental::bfloat16", align 2
```

Comment:
Are names of variable important in the test?

Reply:
Updated names of variable in [a08c68a](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1427/commits/a08c68a6da394e3efed70719dccd9d71d0d679db)